src/unread/unreadSelectors.js

```diff
@@ -155,11 +157,17 @@ export const getUnreadStreamsAndTopics: Selector<$ReadOnlyArray<UnreadStreamItem
         if (!isMuted) {
           total.unread += msgIds.size;
         }
-
+        let isMentioned = false;
+        msgIds.forEach(item => {
+          if (unreadMsgIds.has(item)) {
+            isMentioned = true;
+          }
+        });
```

Comment:
This can be simplified in a couple of ways:
* After we've found one mention, there's no need to look through the rest of the list -- it won't change anything. So it'd be better to break out of the loop there.
  * The `Array#forEach` method doesn't support that. That's one of the reasons it's usually better to use a plain JS for-loop instead; e.g. `for (const item of msgIds) {` rather than `msgIds.forEach(item => {`. Then a `break` statement will end the loop.
* For this use case, there's an `Array` method to do it exactly: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

So:
```suggestion
        const isMentioned = msgIds.some(id => unreadMsgIds.has(id));
```

Then at that point we may as well write it inline in the object literal below, instead of making a variable for it.